demos/storybook/stories/user-menu/within-a-toolbar.stories.ts

```diff
@@ -5,14 +5,17 @@ import { invertRTL } from '../../src/utils';
 export const withinToolbar = (): any => ({
     template: `
         <div>
-            <mat-toolbar style="padding: 0px 16px">
-                <h2>Toolbar Title</h2>
+            <ng-template #avatar>
+                    <mat-icon>person</mat-icon>
+            </ng-template>
+            <mat-toolbar style="padding: 0 16px; height: 4rem;">
+                <h2 class="mat-display-1">Toolbar Title</h2>
                 <pxb-spacer></pxb-spacer>
                 <pxb-user-menu menuTitle="Jane Doe" menuSubtitle="Account Manager" [(open)]="open">
-                    <mat-icon pxb-avatar>person</mat-icon>
-                    <mat-icon pxb-menu-avatar>person</mat-icon>
+                    <template pxb-avatar [ngTemplateOutlet]="avatar"></template>
+                    <template pxb-menu-avatar [ngTemplateOutlet]="avatar"></template>
                     <div pxb-menu-body>
```

Comment:
The `<div style="font-size: 16px; padding: 16px; height: 100px;">Body Content Goes Here</div>` line will appreciate a `mat-body-1` class

Reply:
You got it.